src/common/EmptyView/EmptyView.js

```diff
@@ -13,6 +13,7 @@ const EmptyView = ({
   docsUrl,
   description,
   urlLabel,
+  shouldBeHidden,
```

Comment:
Button shouldn't be render if buttonLabel props is undefined. We don't need new one.